### ISSUE TITLE
The double "onchange" event tests doesn't cover existing value change

### ISSUE THREAD

**Issue by aik099** [minkphp/Mink#529]
When I was writing tests to verify that the `onchange` js event fired only once per `setValue` call (see #518) I've empty inputs and set a value to them. Such approach might not cover all use caseses.

I think, that we need to update tests to cover this:
1. value change from empty to non-empty (already covered)
2. value change non-empty to other non-empty (needs to be covered)
